### PR TITLE
Allow the processing of multiple assays (Issue #175)

### PR DESCRIPTION
**Pull request by cvanderaa**
This PR improves current functionality for QFeatures processing (including imputation and aggregation). It also provides speed improvement by omitting validObject when it is not necessary. Unit test have been adapted as well.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 ## QFeatures 1.9.2
 
-- Nothing yet.
+- feat: aggregateFeatures() can now take multiple assays
+- feat: imputate() can now take multiple assays
```
Comment (lgatto) [inline NEWS.md]: this should be impute()

File: R/QFeatures-class.R [hunk 2]
```diff
@@ -974,6 +969,22 @@ setReplaceMethod("[[", c("QFeatures", "ANY", "ANY", "ANY"),
                      } 
                  })
 
+.normIndex <- function(object, i, allowAbsent = FALSE) {
```
Comment (lgatto) [inline R/QFeatures-class.R:988]: Could you add a comment with a short summary of what that function does.

**Review by lgatto — COMMENTED** [review]
Will look more into it later.

**Comment by cvanderaa** [comment]
Sorry for the delay... I adapted the PR to your comments. Let me know if there is more I should adapt.

**Review by lgatto — APPROVED** [review]

**Pull request merged**